treeherder/perf/migrations/0018_remove_old_frameworks.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 2.2.4 on 2019-09-17 12:23
+
+from django.db import migrations
+
+
+def delete_old_frameworks(apps, schema_editor):
+    PerformanceFramework = apps.get_model('perf', 'PerformanceFramework')
+    frameworks_to_remove = ['talos-aws', 'hasal', 'servo-perf', 'autophone']
+    for framework_to_remove in frameworks_to_remove:
```

Comment:
We could employ a single, chained Django ORM construct, instead of 2 nested for-loops. Using `name__in` would help do that trick.

Reply:
Thanks. Will do.